packages/swing-store/src/internal.js

```diff
@@ -9,6 +9,7 @@ import { Fail, q } from '@agoric/assert';
  *    transcriptStore: TranscriptStoreInternal,
  *    snapStore: SnapStoreInternal,
  *    bundleStore: BundleStoreInternal,
+ *    setUnsafeFastMode: (enabled: boolean) => void,
```

Comment:
This extension makes `yarn lint` complain about the first argument to `assertComplete` no longer satisfying type SwingStoreInternal at https://github.com/Agoric/agoric-sdk/blob/c3682733c0a2fcf19ce13f46819b17f76e8fd8fc/packages/swing-store/src/exporter.js#L111-L112

Maybe modification of SQLite modes should just remain internal to `makeSwingStore`?